src/NuGet.Clients/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -401,6 +400,7 @@ private async Task<IReadOnlyList<RestoreSummary>> PerformNuGetV2RestoreAsync(Pac
                 packageRestoredEvent: (sender, args) => { Interlocked.Add(ref installCount, args.Restored ? 1 : 0); },
                 packageRestoreFailedEvent: (sender, args) => { failedEvents.Enqueue(args); },
                 sourceRepositories: repositories,
+                auditSources: auditSources,
```

Comment:
Are there some tests we can add?

A similar test that might help you figure out the structure and test location: https://github.com/NuGet/NuGet.Client/blob/f61900e4d90c310e0c1615ab8db616cc65810859/test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs#L3242-L3284.

Reply:
I couldn't get the tests to run in Visual Studio or with "dotnet test".

I'll leave this to someone else.